README.md

```diff
@@ -16,6 +16,295 @@ K8ssandra 1.x is configured, packaged, and deployed via Helm charts. Those Helm
 
 K8ssandra 2.x will be based on the this operator.
 
+One of the primary features of this operator is multi-cluster support which will facilitate multi-region Cassandra clusters.
+
+# Installing the operator
+There are a couple of options for installing the operator - build from source or remote installs via kustomize.
+
+**Note:** There are plans to add a Helm chart as well (TODO create and reference ticket here)
+
+**Note:** This section focuses on a single cluster install. See the multi-cluster section below for details on how to configure the operator for a multi-cluster install.
+
+## Remote Install
+Kustomize supports building resources with remote URLs. See this Kustomize [doc](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md) for background.
+
+This section provides some examples that demonstrate how to configure the operator installation via Kustomize.
+
+You need to Kustomize 4.0.5 or later installed. See [here](https://kubectl.docs.kubernetes.io/installation/kustomize/) for installation options. 
+
+Recent versions of `kubectl` include Kustomize. It is executed using the `-k` option. I prefer to install Kustomize and use the `kustomize` binary as I have found in the past that the one embedded with `kubectl` can be several versions behind and behave differently  than what is described in the Kustomize docs.
```

Comment:
This has burnt me in the past. I believe the situation is better now RE kustomize version recency, but it is good to note that this can be an issue.